dragn/dice/__init__.py

```diff
@@ -1,6 +1,26 @@
+import re
+from typing import Any
+
 from dragn.dice.die import DieBuilder
 
 
+def __getattr__(name: str) -> Any:
+    """
+    Module level __getattr__ to automatically create dice when
+    one a die of non-standard sizes.
+    """
+    m = re.match(r"D(\d+)", name)
+    # Avoiding the Walrus operator here for backwards compatibility :(
```

Comment:
No need to have sadness in our code, when python 3.8 is released, we can use it, and we'll remember it, no need for a comment :).